8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java

```diff
@@ -45,6 +55,9 @@
     private Button yellow_button;
     private Button blue_button;
 
+    private ConstraintLayout constraintLayout_select_color;
+
+    @SuppressLint("ClickableViewAccessibility")
```

Comment:
This particular suppressLint is allowed by the way, will add it to the lint config, so rule sticks around. Don't use SuppressLint annotation.